Core/src/org/sleuthkit/autopsy/discovery/ui/MiniTimelineArtifactListPanel.java

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
It looks like this file was generated with the gui designer.  Is there are corresponding .form file?